packages/web/package.json

```diff
@@ -0,0 +1,157 @@
+{
+  "NOTE": "This package.json should be in sync with @draftbit/ui's package.json",
```

Comment:
I think having to manually sync these is going to be almost as bad as a mutat-ey script.

Any way to avoid that or reduce that? For example, have this depend on @draftbit/ui or a @draftbit/core package?

Reply:
The only way I thought, if we want to keep the dependency graph is, to generate both package.jsons from a template. This would solve the repetition but would add another layer of complex.

Reply:
Another approach, but this would require more changes, could be to keep web/ui part of each component and have a change on the build tool where are both exposed under the same package.